tests/gis_tests/layermap/models.py

```diff
@@ -92,3 +97,8 @@ class Invalid(models.Model):
                  'length': 'Length',
                  'path': 'LINESTRING',
                  }
+province_mapping = {'name': 'provincien',
+                    'bag_id': 'bag_id',
+                    'border': 'MULTIPOLYGON'
```

Comment:
Include a trailing comma in cases like this so that if more items are added later, this line doesn't have to be modified again.